docs/04-operation-guides/troubleshooting/apix-02-401-unauthorized-403-forbidden.md

```diff
@@ -0,0 +1,93 @@
+      ```
+
+  2. Encode your client credentials and export them as an environment variable:
+
+      ```bash
+      export ENCODED_CREDENTIALS=$(echo -n "$CLIENT_ID:$CLIENT_SECRET" | base64)
+      ```
+
+  3. Check the access token status:
+
+      ```bash
+      curl -X POST "https://oauth2.{DOMAIN}/oauth2/introspect" -H "Authorization: Basic $ENCODED_CREDENTIALS" -F "token={ACCESS_TOKEN}"
+      ```
+
+  4. Generate a [new access token](../../03-tutorials/00-api-exposure/apix-03-expose-and-secure-service.md#register-an-oauth2-client-and-get-tokens) if needed.
+
+- Your client from the OAuth2Client resource is registered properly in Hydra OAuth2 and the OpenID Connect server. You need to call the Hydra administrative endpoint `/client` from inside of the cluster. Follow these steps:
+
+  1. Fetch the Client ID from the Secret specified in the OAuth2Client resource:
+
+      ```bash
+      kubectl get secrets {SECRET_NAME} -n {SECRET_NAMESPACE} -o jsonpath='{ .data.client_id }' | base64 --decode
+      ```
+
+  2. Create a simple curl Pod:
+
+      ```bash
+      cat <<EOF | kubectl apply -f -
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        labels:
+          app: ory-curl
+        name: ory-curl
+        namespace: {SECRET_NAMESPACE}
+      spec:
+        containers:
+        - name: curl
+          image: alpine
+          terminationMessagePolicy: "FallbackToLogsOnError"
+          command:
+            - /bin/sh
+            - -c
+            - |
+              apk add curl jq
+              curl ory-hydra-admin.kyma-system.svc.cluster.local:4445/clients | jq '.'
+      EOF
+      ```
+
+  3. Check logs from the `ory-curl` Pod:
+
+      ```bash
+      kubectl logs -n {SECRET_NAMESPACE} ory-curl curl
+      ```
+
+  4. If the Client ID from step 1 is not available on the client list, make sure Hydra has access to the database and/or restart the Hydra Measter Pod. You can check the logs using the following commands:
+
+     ```bash
+      # Check logs from the Hydra-Maester controller application
+      kubectl logs -n kyma-system -l "app.kubernetes.io/name=hydra-maester" -c hydra-maester
+      # Example output
+      2020-05-04T12:19:04.472Z  INFO  controller-runtime.controller Starting EventSource  {"controller": "oauth2client", "source": "kind source: /, Kind="}2020-05-04T12:19:04.472Z  INFO  setup starting manager
+      2020-05-04T12:19:04.573Z  INFO  controller-runtime.controller Starting Controller {"controller": "oauth2client"}
+      2020-05-04T12:19:04.673Z  INFO  controller-runtime.controller Starting workers  {"controller": "oauth2client", "worker count": 1}
+      2020-05-04T12:26:30.819Z  INFO  controllers.OAuth2Client  using default client
+      2020-05-04T12:26:30.835Z  INFO  controllers.OAuth2Client  using default client
+      # This log entry informs that a client has been created, and should be visible within hydra
```

Comment:
```suggestion
      # This log entry informs us that a client has been created, and should be visible within Hydra
```
Sorry, I missed this one before. This is minor, but `inform` is a transitive verb and needs an object (`us`/`you`). See the [Cambridge Dictionary](https://dictionary.cambridge.org/dictionary/english/inform) for reference.